test/jdk/java/nio/channels/Channels/TransferTo.java

```diff
@@ -141,24 +141,29 @@ public void testMoreThanTwoGB() throws IOException {
         // preparing two temporary files which will be compared at the end of the test
         Path sourceFile = Files.createTempFile(null, null);
         Path targetFile = Files.createTempFile(null, null);
```

Comment:
can we add an identifier to the tmp files ? In the case where deletion fails, we'll be able to identify the origin more easily.